mir-module/src/main/resources/META-INF/resources/js/mir/accesskey.js

```diff
@@ -0,0 +1,244 @@
+const RSC_URL = webApplicationBaseURL + "rsc/accesskey/";
+let labels = undefined;
+
+function getParameterByName(name, url = window.location.href) {
+    name = name.replace(/[\[\]]/g, '\\$&');
+    var regex = new RegExp('[?&]' + name + '(=([^&#]*)|&|#|$)'),
+    results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCoRe-Org/mir/pullRequest?prid=7162253)
```suggestion
    if (!results[2]) return "";
```